### ISSUE TITLE
At train AnoGAN with MNIST data, Does the train set and test set all contain only normal data?

### ISSUE THREAD

**Issue by bemoregt** [tkwoo/anogan-keras#1]
Hi, @tkwoo 
 
In the case of Anomaly Gan, I knew that it trained with only normal datasets.

At train AnoGAN with MNIST data, Does the train set and test set all contain only normal data?

Or are both normal + abnormal divided into folders? 
I wonder.

form @bemoregt.

**Comment by tkwoo** [comment]
Train sets contain only '1' class in this repo.
I also use test set of '1' because want more large size of dataset
